mu/srv/routes.json

```diff
@@ -0,0 +1,10 @@
+{
+    "sim_start": "sim/start",
+    "sim_stop": "sim/stop",
+    "sim_cat": "sim/cat",
+    "sim_list": "sim/list",
+
+    "logs": "logs",
+
+    "reset": "reset"
+}

```

Comment:
true on using `POST` but I think I'll leave it out for now since that's a readability / understandability change that's not super important right now since everything's just using query parameters. might have to figure out request bodies in the next part though so this might change